scripts/build.sh

```diff
@@ -60,8 +60,18 @@ rm -f bin/*
 mkdir -p bin/
```

Comment:
These `rm` and `mkdir` should be updated to use `${BIN_PARENT_DIR}`, so you might want to move that logic up.